src/components/existing-request/existing-request-display.vue

```diff
@@ -2,12 +2,24 @@
   <MainContainer id="existing-request-display" class="pa-10">
     <template v-slot:container-header>
       <v-col cols="auto" class="py-0">
-        <span class="h3 user-select-all">{{ nr.nrNum }}</span>
-        <span class="h6 ml-4">{{ entityTypeCdToText(nr.entity_type_cd) }}</span>
+        <span v-if="!isIncompletePayment" class="h3 user-select-all mr-4">{{ nr.nrNum }}</span>
+        <span class="h6">{{ entityTypeCdToText(nr.entity_type_cd) }}</span>
       </v-col>
     </template>
 
     <template v-slot:content>
+      <div v-if="isIncompletePayment" style="padding: 25px 0px 16px 0px">
```

Comment:
Please use [Vuetify spacing helpers](https://vuetifyjs.com/en/styles/spacing/) here instead of inline styles.

Reply:
Thanks!